src/store/accounts/api.ts

```diff
@@ -0,0 +1,143 @@
+import { atomFamilyWithInfiniteQuery } from 'jotai-query-toolkit';
+import { atomFamilyWithQuery } from '@store/query';
+import { QueryRefreshRates } from '@common/constants';
+import { apiClientState } from '@store/common/api-clients';
+import type {
+  MempoolTransaction,
+  Transaction,
+  AccountDataResponse,
+} from '@stacks/stacks-blockchain-api-types';
+import type {
+  AccountBalanceResponseBigNumber,
+  AddressBalanceResponse,
+  AccountBalanceStxKeys,
+} from './types';
+
+import BigNumber from 'bignumber.js';
+import { atomFamily } from 'jotai/utils';
+import { atom } from 'jotai';
+import { AccountStxBalanceBigNumber } from './types';
+
+enum AccountClientKeys {
+  InfoClient = 'account/InfoClient',
+  BalancesClient = 'account/BalancesClient',
+  TransactionsClient = 'account/TransactionsClient',
+  MempoolTransactionsClient = 'account/MempoolTransactionsClient',
+}
+
+type PrincipalWithNetworkUrl = { principal: string; networkUrl: string };
+type PrincipalWithLimitNetworkUrl = { principal: string; limit: number; networkUrl: string };
+
+export interface ResultsWithLimitOffsetTotal<T> {
```

Comment:
`PaginatedResults<>` would be a better name imo, though can't you pull this from the types repo? 🤔

Reply:
I tried to, but it's not available. Most of the types are incorrect, as it's `results: object` lol